src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs

```diff
@@ -70,7 +70,38 @@ public override Task<ExportResult> ExportAsync(IEnumerable<Activity> activityBat
                         System.Console.WriteLine("Activity Tags");
                         foreach (var tag in activity.TagObjects)
                         {
-                            System.Console.WriteLine($"\t {tag.Key} : {tag.Value}");
+                            if (tag.Value is int[] intArray)
+                            {
+                                foreach (var item in intArray)
+                                {
+                                    System.Console.WriteLine($"\t {tag.Key} : {item}");
+                                }
+                            }
+                            else if (tag.Value is string[] stringArray)
```

Comment:
I think the key here is that we don't have to introduce if-else for different type of arrays.
Could we cast the object to IEnumerable?

Reply:
I think so. Below what I did:
```csharp
if (tag.Value.GetType().IsArray)
{
foreach (var item in ((IEnumerable)tag.Value).Cast<object>())
{
    System.Console.WriteLine($"\t {tag.Key} : {item}");
}
}
else
{
System.Console.WriteLine($"\t {tag.Key} : {tag.Value}");
}
```

Reply:
This is getting better, thanks.
On the correctness side: if the type is array (e.g. `[1,2,3]`), we probably want to have something like:
```text
    Foo: [1, 2, 3]
```
instead of:
```text
    Foo: 1
    Foo: 2
    Foo: 3
```
This is not blocking and you can leave it to @cijothomas (the current ConsoleExporter default output looks awful today and we need to redo it).